meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -114,6 +114,17 @@ export class MongoEntityConverter {
 		return mongoRundowns.map(this.convertRundown)
 	}
 
+	public convertToMongoRundown(rundown: Rundown): Partial<MongoRundown> {
+		return {
+			_id: rundown.id,
+			name: rundown.name,
+		}
+	}
+
+	public convertToMongoRundowns(rundowns: Rundown[]): Partial<MongoRundown>[] {
+		return rundowns.map(this.convertToMongoRundown)
```

Comment:
To ensure correct behaviour, the passed method should be bound to the class instance like so:
```suggestion
		return rundowns.map(this.convertToMongoRundown.bind(this))
```

This applies to other places as well.

Reply:
Applied the suggested change to all similar method calls in the same file.

Reply:
Or do `rundown.map(rundown => this.convertToMongoRundown(rundown))`

Pick whatever you see fit :)